client/src/components/overview/GalleryView.jsx

```diff
@@ -85,12 +82,15 @@ function findAverageRating(ratings)  {
 
 // This is the actual functional component
 function Overview(props) {
+  var [expanded, setView] = useState(false);
+  var [imgIndex, setImgIndex] = useState(0);
   const [styleIndex, setStyleIndex] = useState(0)
```

Comment:
semicolon

Reply:
Yeah. Lots of semicolons missing this pull.